Branch-SDK/src/main/java/io/branch/referral/ServerRequestGetCPID.java

```diff
@@ -2,6 +2,8 @@
 
 import android.content.Context;
 
+import androidx.annotation.NonNull;
```

Comment:
Does this mean this version of the SDK will require androidx? Does that indicate a new major revision?

Reply:
hmm, androidx was introduced in v4.0.0 though. Has this been causing problems to React Native < 0.60? I though it was working and we already made a release with v4.3.2?